src/TimerRecordDialog.cpp

```diff
@@ -591,9 +591,21 @@ int TimerRecordDialog::ExecutePostRecordActions(bool bWasStopped) {
 
    // Do Automatic Export?
    if (m_bAutoExportEnabled) {
+      const auto& tracks = TrackList::Get(mProject);
       Exporter e{ mProject };
+      
+      bool skipSilenceAtBeginning;
+      gPrefs->Read(wxT("/AudioFiles/SkipSilenceAtBeginning"),
+                                      &skipSilenceAtBeginning, false);
+      
+      if(!e.SetExportRange(0, tracks.GetEndTime(), false, skipSilenceAtBeginning))
+      {
+         //But there should be at least one recorded track...
+         ShowExportErrorDialog(
+            ":576", XO("All audio is muted."), XO("Warning"), false);
```

Comment:
You lifted this UI out of the testing logic,but now it duplicates a string literal in two places.  Is there a good file in which to put a single occurrence of the literal?

Reply:
No, but if I understand correctly this really shouldn't happen.